src/language/mongoDBService.ts

```diff
@@ -965,6 +1060,7 @@ export default class MongoDBService {
       this._provideIdentifierObjectValueCompletionItems.bind(this, state),
       this._provideTextObjectValueCompletionItems.bind(this, state),
       this._provideCollectionSymbolCompletionItems.bind(this, state),
+      this._provideStreamProcessorSymbolCompletionItems.bind(this, state),
```

Comment:
Kind of a broader question, would it make sense to conditionally set the stream processor completion items and only run the visitor checks when we're connected to an atlas stream? Is there any reason we want these completion items for normal connections?

Reply:
I think generally we don't disable things in shell. For example, `db` can still be used even if we are connected to an Atlas Stream Processing and `sp` is also there when connected to a normal Mongo Cluster. This will be an empty array so shouldn't have any effect.